Fabric/src/main/java/at/petrak/hexcasting/fabric/mixin/FabricPlayerMixin.java

```diff
@@ -22,5 +22,9 @@ protected FabricPlayerMixin(EntityType<? extends LivingEntity> entityType, Level
         var out = cir.getReturnValue();
         out.add(HexAttributes.GRID_ZOOM);
         out.add(HexAttributes.SCRY_SIGHT);
+        out.add(HexAttributes.FEEBLE_MIND);
+        out.add(HexAttributes.MEDIA_CONSUMPTION_MODIFIER);
+        out.add(HexAttributes.AMBIT_RADIUS);
+        out.add(HexAttributes.SENTINEL_RADIUS);
```

Comment:
Should these be like,, auto grabbed out of the HexAttributes instead of manually listed in 3 separate places?